Domain.Tests/EventSourcedRepositoryTests.cs

```diff
@@ -624,6 +627,37 @@ public void When_Refresh_is_called_on_an_aggregate_having_pending_events_it_thro
             var order = await repository.GetLatest(orderId);
 
             order.CustomerName.Should().Be("Waylon Jennings");
+        }
+
+
+        [Category("Performance")]
+        [Test, Timeout(120000)]
+        public async Task When_sourcing_an_aggregate_with_a_large_number_of_source_events_then_the_operation_completes_quickly()
+        {
+            var count = 30000;
+            var aggregateId = Guid.Parse("547E3646-DBE5-43D4-BAC9-E391336340D7");
```

Comment:
I think it's better to use a random guid. Otherwise, the behavior of the test varies based on the state from previous test runs.